backend/prompt_studio/prompt_studio_v2/models.py

```diff
@@ -84,6 +92,13 @@ class Mode(models.TextChoices):
         db_comment="Field to store the prompt key",
         unique=False,
     )
+    required = models.CharField(
```

Comment:
@vishnuszipstack please add a `db_comment` specifying that it will be used for HQR to mark necessary fields. Just `required` doesn't convey this clearly and it'll not be understandable for people looking at the code without this context

Reply:
done